### PR TITLE
feat: add integration tests for users api `AP-1239`

### PR DESCRIPTION
**Pull request by BryanttV**
## Description
This PR adds an integration test for the users' API. This test is executed in the job of **Tutor Integration Tests**

## Testing instructions

Check the jobs of **Tutor Integration Tests** in the PR.
## Additional Information
The Palm Release (`<17.0.0`) was removed from the integration tests workflow.
## Jira Issue

- [AP-1239](https://edunext.atlassian.net/browse/AP-1239)

[AP-1239]: https://edunext.atlassian.net/browse/AP-1239?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: eox_core/tests/tutor/integration_test_users.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+    def setUp(self):
+        """Set up the test suite"""
+        self.path = reverse("eox-core:eox-api:eox-api:edxapp-user")
+        self.admin_user = create_admin_user()
+        create_oauth2_client(self.admin_user)
+        self.tenant_x = self.create_tenant("tenant-x")
+        self.tenant_y = self.create_tenant("tenant-y")
+
+    @override_settings(EOX_CORE_USERS_BACKEND="eox_core.edxapp_wrapper.backends.users_m_v1")
```
Comment (bra-i-am) [inline eox_core/tests/tutor/integration_test_users.py:65]: can the path be saved in a variable like CLIENT_ID and replaced here and after?

**Review by bra-i-am — COMMENTED** [review]
hey @BryanttV, thanks for this. I only have a little comment, everything else looks good to me. After your answer, I'll give the approval

File: eox_core/tests/tutor/integration_test_users.py (hunk 1 above)
Comment (BryanttV) [reply eox_core/tests/tutor/integration_test_users.py]: You're right!, done: https://github.com/eduNEXT/eox-core/commit/5ca9726d6a2b9214bf50a158d8054282fa2a2e6b

**Review by bra-i-am — APPROVED** [review]
LGTM!

**Review by dcoa — DISMISSED** [review]
LGTM

File: eox_core/api/v1/tests/integration/test_views.py [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+            "fullname": fake.unique.name(),
+            "password": fake.unique.password(),
+            "activate_user": True,
+        }
+        for _ in range(100)
+    ]
+)
+
+
+class BaseAPIIntegrationTest(APITestCase):
```
Comment (magajh) [inline eox_core/api/v1/tests/integration/test_views.py:36]: @BryanttV since we are not using the API client to test the APIs, I think we should just use the TestCase class instead

File: eox_core/api/v1/tests/integration/test_views.py [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+        return response
+
+
+@ddt
+class TestUsersAPIIntegration(BaseAPIIntegrationTest):
+    """Integration test suite for the Users API"""
+
+    def setUp(self):
+        """Set up the test suite"""
+        self.user_url = "/eox-core" + reverse("eox-api:eox-api:edxapp-user")
```
Comment (magajh) [inline eox_core/api/v1/tests/integration/test_views.py:130]: @BryanttV, since we will be running the tests outside the LMS container in an isolated environment, we won’t need to use reverse

File: fixtures/initial_data.json [hunk 4]
```diff
@@ -0,0 +1,137 @@
+[
+    {
+        "model": "sites.site",
+        "pk": 8,
+        "fields": {
+            "domain": "tenant-x.local.edly.io",
```
Comment (magajh) [inline fixtures/initial_data.json:6]: Do we really need more than one tenant (site, route, tenant configt) to test the users API?

**Review by magajh — COMMENTED** [review]
@BryanttV, I’ll leave my full review later, but for now, I’m leaving a few quick comments with some questions and observation

File: fixtures/initial_data.json (hunk 4 above)
Comment (BryanttV) [reply fixtures/initial_data.json:6]: The idea is to check that a user only exists in X tenant, and not in Y.

File: eox_core/api/v1/tests/integration/test_views.py (hunk 3 above)
Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py:130]: I think one of the purposes of using `reverse` is in case the URL changes at some point. But, we can directly define the string.

File: eox_core/api/v1/tests/integration/test_views.py (hunk 2 above)
Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py]: Done: https://github.com/eduNEXT/eox-core/commit/f82e677570ec7bb9bb7b4fbb1f28a090d43db807

File: eox_core/api/v1/tests/integration/test_views.py (hunk 3 above)
Comment (magajh) [reply eox_core/api/v1/tests/integration/test_views.py:130]: Mmm, that's a fair point. Let's try and see if it works with reverse then

**Review by mariajgrimaldi — CHANGES_REQUESTED** [review]
Thank you both, @BryanttV @magajh!

File: eox_core/api/v1/tests/integration/test_views.py [hunk 5]
```diff
@@ -0,0 +1,384 @@
...
+
+    # pylint: disable=too-many-arguments
+    def make_tenant_request(
+        self,
+        tenant: dict,
+        method: str,
+        url: str,
+        data: dict | None = None,
+        params: dict | None = None,
+    ) -> requests.Response:
```
Comment (mariajgrimaldi) [inline eox_core/api/v1/tests/integration/test_views.py:94]: I'm sorry, I forgot to leave this comment. 

This method assumes we're working with tenants, but that's not always the case. Should we call it `make_request` instead?

File: eox_core/api/v1/tests/integration/test_views.py [hunk 6]
```diff
@@ -0,0 +1,388 @@
...
+        return response
+
+
+@ddt
+class TestUsersAPIIntegration(BaseAPIIntegrationTest):
+    """Integration test suite for the Users API"""
+
+    def setUp(self):
+        """Set up the test suite"""
+        self.user_url = "/eox-core" + reverse("eox-api:eox-api:edxapp-user")
```
[the remark this answers is not in the document]
Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py]: Done

File: eox_core/api/v1/tests/integration/test_views.py [hunk 7]
```diff
@@ -0,0 +1,388 @@
...
+
+        method = method.upper()
+        if method == "POST":
+            response = requests.post(full_url, data=data, headers=headers, timeout=API_TIMEOUT)
+        elif method == "GET":
+            response = requests.get(full_url, params=params, headers=headers, timeout=API_TIMEOUT)
+        elif method == "PATCH":
+            response = requests.patch(full_url, json=data, headers=headers, timeout=API_TIMEOUT)
+        else:
+            raise ValueError(f"Unsupported HTTP method: {method}")
```
[the remark this answers is not in the document]
Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py]: Done

File: eox_core/api/v1/tests/integration/test_views.py [hunk 8]
```diff
@@ -0,0 +1,388 @@
...
+
+        Expected result:
+        - The status code is 404.
+        - The response contains an error message.
+        """
+        response = self.get_user_in_tenant(self.tenant_x, {param: value})
+
+        response_data = response.json()
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+        self.assertIn("detail", response_data)
```
[the remark this answers is not in the document]
Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py:280]: Yes, it would be convenient. Done

File: eox_core/api/v1/tests/integration/test_views.py [hunk 9]
```diff
@@ -0,0 +1,388 @@
...
+
+        Expected result:
+        - The status code is 404.
+        - The response contains an error message.
+        """
+        response = self.update_user_in_tenant(self.tenant_x, {param: value})
+
+        response_data = response.json()
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+        self.assertIn("detail", response_data)
```
[the remark this answers is not in the document]
Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py:355]: Done

File: eox_core/api/v1/tests/integration/test_views.py [hunk 10]
```diff
@@ -0,0 +1,384 @@
...
+        Args:
+            tenant (dict): The tenant data.
+            user_data (dict): The user data.
+
+        Returns:
+            requests.Response: The response object.
+        """
+        return self.make_tenant_request(tenant, "PATCH", url=self.user_updater_url, data=user_data)
+
+    def test_create_user_in_tenant_success(self) -> None:
```
[the remark this answers is not in the document]
Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py:171]: I was checking and apparently, the endpoint doesn't receive fields to assign permissions (`is_staff` or `is_superuser`). However, we could check that even if such fields are sent, no additional permissions will ever be assigned to the user. What do you think?

File: eox_core/settings/integration_test.py [hunk 11]
```diff
@@ -0,0 +1,24 @@
+"""Integration test settings."""
```
[the remark this answers is not in the document]
Comment (BryanttV) [reply eox_core/settings/integration_test.py]: I deleted this file.

File: requirements/base.txt
[the remark this answers is not in the document]
Comment (BryanttV) [reply requirements/base.txt:1]: For the new `Faker` package

File: fixtures/initial_data.json [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+            "authorization_grant_type": "client-credentials",
+            "client_secret": "client_secret",
+            "name": "eox-core-app",
+            "skip_authorization": false,
+            "created": "2024-09-06T13:30:17.347Z",
+            "updated": "2024-09-06T13:30:17.347Z",
+            "algorithm": ""
+        }
+    }
+]
```
[the remark this answers is not in the document]
Comment (BryanttV) [reply fixtures/initial_data.json:137]: Done

File: eox_core/settings/integration_test.py [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+
+# pylint: disable=wildcard-import
+from edx_django_utils.plugins import add_plugins
+
+try:
+    from lms.envs.common import *
```
[the remark this answers is not in the document]
Comment (BryanttV) [reply eox_core/settings/integration_test.py]: I deleted this file. Thanks

File: eox_core/api/v1/tests/integration/test_views.py (hunk 5 above)
Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py]: Yes, you're right. I updated the method name.

File: eox_core/api/v1/tests/integration/test_views.py [hunk 14]
```diff
@@ -0,0 +1,388 @@
...
+        {
+            "username": fake.unique.user_name(),
+            "email": fake.unique.email(),
+            "fullname": fake.unique.name(),
+            "password": fake.unique.password(),
+            "activate_user": True,
+        }
+        for _ in range(100)
+    ]
+)
```
[the remark this answers is not in the document]
Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py:34]: 1. We could generate only the necessaries. I will try it.
2. Actually, I don't think the data would cause any errors. Another option would be to have a JSON file with the user's data to create, but we should update it when more tests are added. What do you think?

File: scripts/integration.sh [hunk 15]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (BryanttV) [reply scripts/integration.sh:1]: Yes, I agree. The tasks to be performed in all plugins should be left in the action and avoid repetition.

File: eox_core/api/v1/tests/integration/test_views.py (hunk 10 above)
[the remark this answers is not in the document]
Comment (mariajgrimaldi) [reply eox_core/api/v1/tests/integration/test_views.py]: That sounds like a good test case to check, so yes, go ahead. Thank you!

File: eox_core/settings/integration_test.py (hunk 11 above)
[the remark this answers is not in the document]
Comment (mariajgrimaldi) [reply eox_core/settings/integration_test.py]: Don't we need a configuration file for tests?

File: eox_core/api/v1/tests/integration/test_views.py (hunk 14 above)
[the remark this answers is not in the document]
Comment (mariajgrimaldi) [reply eox_core/api/v1/tests/integration/test_views.py:34]: 1. Thanks!
2. You're right; tests should be idempotent, so it shouldn't matter whether the data changes. We can go ahead and use these for better coverage. We can revisit it if we have issues later on.

[the remark this answers is not in the document]
Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py]: Done.

File: eox_core/api/v1/tests/integration/test_views.py (hunk 10 above)
[the remark this answers is not in the document]
Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py]: Done

File: eox_core/settings/integration_test.py (hunk 11 above)
[the remark this answers is not in the document]
Comment (BryanttV) [reply eox_core/settings/integration_test.py:1]: These settings were directly in the `test_views.py` file, but I moved them to the `integration_test.py` file. What do you think?
https://github.com/eduNEXT/eox-core/blob/38ac0c2c299e9b2965f0d8026efb5ae88db90358/eox_core/settings/integration_test.py#L3-L7

File: eox_core/settings/integration_test.py (hunk 13 above)
[the remark this answers is not in the document]
Comment (BryanttV) [reply eox_core/settings/integration_test.py]: I added the file again, but removed the settings from the LMS

File: eox_core/api/v1/tests/integration/test_views.py [hunk 16]
```diff
@@ -0,0 +1,402 @@
...
+            # "bio": fake.text(),
+        }
+
+        response = self.update_user_in_tenant(self.tenant_x, user_data=updated_data)
+
+        response_data = response.json()
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response_data["username"], data["username"])
+        self.assertEqual(response_data["email"], data["email"])
+        self.assertEqual(response_data["name"], updated_data["fullname"])
```
Comment (magajh) [inline eox_core/api/v1/tests/integration/test_views.py:339]: I was thinking about suggesting a loop or some way to automate these field assertions, but then I noticed there’s a discrepancy between the field name in Open edX and what our API returns. So, I’m on board with this approach

**Review by magajh — COMMENTED** [review]
Thanks, @BryanttV!

**Comment by magajh** [comment]
@BryanttV, tested the changes from this PR in [another branch](https://github.com/eduNEXT/eox-core/pull/278) with the [workflow](https://github.com/eduNEXT/integration-test-in-tutor/pull/4) that executes the tests outside the container and It worked just fine, thanks for your help!

Here are the changes I did in the branch to make it work:
Script to run the tests:
```
#!/bin/bash

bgreen () { printf "\n\n\e[1m\e[32m" ; $@ ; printf "\e[0m\n\n"; }

bgreen echo "Install tests requirements"
make install-dev-dependencies

bgreen echo "Run integration tests"
pytest eox_core/api/v1/tests/integration/test_views.py

```

Workflow:
```
name: Tutor Integration Tests
on: [pull_request]

jobs:
  integration-test:
    name: Tutor Integration Tests
    runs-on: ubuntu-latest
    strategy:
      matrix:
        tutor_version: ['<18.0.0', '<19.0.0']
    steps:
      - name: Run Integration Tests
        uses: eduNEXT/integration-test-in-tutor@mjh/run-integration-tests-outside-container
        with:
          tutor_version: ${{ matrix.tutor_version }}
          app_name: 'eox-core'
          openedx_extra_pip_requeriments: 'eox-tenant'
          shell_file_to_run: 'scripts/integration.sh'
          fixtures_file: 'fixtures/initial_data.json'

```

File: eox_core/api/v1/tests/integration/test_views.py (hunk 14 above)
[the remark this answers is not in the document]
Comment (mariajgrimaldi) [reply eox_core/api/v1/tests/integration/test_views.py]: What @magajh mentioned [here](https://github.com/eduNEXT/eox-core/pull/275#discussion_r1763022408) made me question why we'd need this generator. If test data shouldn't make tests fail and they should be idempotent, why do we need a generator? Can't we manually generate the number of inputs we need, considering edge cases, etc?

File: scripts/integration.sh (hunk 15 above)
[the remark this answers is not in the document]
Comment (mariajgrimaldi) [reply scripts/integration.sh:1]: Did we remove some lines from the bash script to the action itself?

File: setup.cfg [hunk 17]
```diff
@@ -7,10 +7,11 @@ tag = False
 
 [tool:pytest]
 DJANGO_SETTINGS_MODULE = eox_core.settings.test
+addopts = --ignore-glob='**/integration/*'
```
[the remark this answers is not in the document]
Comment (mariajgrimaldi) [reply setup.cfg:10]: Should we have an entry in the makefile to handle integration tests? So the `scripts/integration` calls it instead. Then, it'd be easier for those executing tests locally to know which command to use.

File: scripts/integration.sh (hunk 15 above)
[the remark this answers is not in the document]
Comment (magajh) [reply scripts/integration.sh:1]: Hey @mariajgrimaldi @BryanttV,

With the changes made to the action [here](https://github.com/eduNEXT/integration-test-in-tutor/pull/4), we now only need to install the test requirements (which may vary from one plugin to another) and run pytest. 

For example:
```bash
#!/bin/bash

bgreen () { printf "\n\n\e[1m\e[32m" ; $@ ; printf "\e[0m\n\n"; }

bgreen echo "Installing test requirements"
make install-dev-dependencies

bgreen echo "Running integration tests"
pytest eox_core/api/v1/tests/integration/test_views.py
```

If you think there are other steps that should be the responsibility of the action, please let me know in the PR for the action. Let’s continue that conversation there.

File: setup.cfg (hunk 17 above)
[the remark this answers is not in the document]
Comment (magajh) [reply setup.cfg:10]: @mariajgrimaldi that's a great idea

File: scripts/integration.sh
Comment (magajh) [inline scripts/integration.sh:1]: @BryanttV can we please make the name of this script a bit more specific? what do you think of something like execute_integration_tests.sh

File: .github/workflows/integration-test.yml [hunk 18]
```diff
@@ -7,13 +7,21 @@ jobs:
...
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout Code
+        uses: actions/checkout@v4
         with:
-          path: eox-core
-      - uses: eduNEXT/integration-test-in-tutor@main
+          path: 'eox-core'
+
+      - name: Print Current Directory
```
[the remark this answers is not in the document]
Comment (BryanttV) [reply .github/workflows/integration-test.yml]: Done: https://github.com/eduNEXT/eox-core/commit/a24c620d0731dc62a2066a8a978dda0b9fffd258

File: scripts/integration.sh [hunk 19]
```diff
@@ -0,0 +1,19 @@
...
+
+bgreen () { printf "\n\n\e[1m\e[32m" ; $@ ; printf "\e[0m\n\n"; }
+
+# This script installs the package in the edxapp environment, installs test
```
[the remark this answers is not in the document]
Comment (BryanttV) [reply scripts/integration.sh]: Done: https://github.com/eduNEXT/eox-core/commit/365d09bb8953a26e1d0cadb861307568d2ff3654

File: scripts/integration.sh
Comment (BryanttV) [reply scripts/integration.sh:1]: Done: https://github.com/eduNEXT/eox-core/commit/322d0e66de8d025df267f983f4e69861458b3ae8

File: setup.cfg (hunk 17 above)
[the remark this answers is not in the document]
Comment (BryanttV) [reply setup.cfg]: @magajh @mariajgrimaldi, thanks for your suggestions!
I simplified the script file, including the new make command, to run integration tests. What do you think? https://github.com/eduNEXT/eox-core/commit/fd0d697fa7bc1574c366048f39ed251974cf61d7

File: eox_core/api/v1/tests/integration/test_views.py [hunk 20]
```diff
@@ -0,0 +1,402 @@
...
+from ddt import data as ddt_data
+from ddt import ddt, unpack
+from django.test import TestCase
+from django.urls import reverse
+from faker import Faker
+from rest_framework import status
+
+from eox_core.settings.integration_test import API_TIMEOUT, CLIENT_ID, CLIENT_SECRET, EOX_CORE_API_BASE, LMS_BASE
+
+fake = Faker()
```
[the remark this answers is not in the document]
Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py:17]: You mentioned the main reasons for using Faker, besides having test data available automatically without manually doing it every time we want to include a new test.

However, we can include a static dataset, perhaps a JSON file, or simply a variable in another file containing such data, which could be loaded when running the tests.

What do you think? Do you have any other ideas?

File: eox_core/settings/integration_test.py (hunk 11 above)
[the remark this answers is not in the document]
Comment (BryanttV) [reply eox_core/settings/integration_test.py]: Thanks @magajh, I included the changes with your suggestion: https://github.com/eduNEXT/eox-core/commit/711e03fe769bbf19e41aba47f4667581ec3b3cce

File: Makefile [hunk 21]
```diff
@@ -49,6 +49,9 @@ python-quality-test:
 
 run-tests: python-test python-quality-test
 
+run-integration-tests: install-dev-dependencies
```
Comment (magajh) [inline Makefile:52]: Nice! I think this is ok for now, once we've moved all the unit tests to their respective unit folders,we can change this to do something like pytest ./eox_core --ignore-glob='**/unit/*'

Comment (magajh) [inline Makefile:52]: @BryanttV, I see that you've moved all the tests into their respective unit folders. I'm guessing we're not using something like `pytest ./eox_core --ignore-glob='**/unit/*'` because the integration tests for the backend imports would fail, right?

Comment (BryanttV) [reply Makefile:52]: Yes, and also for the tests inside `eox_core/tests`. But what do you think about this change? We would not need to modify the command as long as the new tests are inside an `integration` folder.

https://github.com/eduNEXT/eox-core/commit/74a04f5358c97c11afb7b3d48aa184de36eea654

File: eox_core/api/v1/tests/integration/test_views.py (hunk 14 above)
[the remark this answers is not in the document]
Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py]: Yes, we could do that. We can remove the Faker library and include static data for users. What do you think? @magajh @mariajgrimaldi

File: Makefile (hunk 21 above)
Comment (magajh) [reply Makefile:52]: @BryanttV nice, I agree with those changes. We can leave the `--ignore-glob='**/edxapp_wrapper/*'` while we figure out what to do with the edxapp_wrapper test

File: eox_core/api/v1/tests/integration/test_views.py (hunk 20 above)
[the remark this answers is not in the document]
Comment (mariajgrimaldi) [reply eox_core/api/v1/tests/integration/test_views.py:16]: I think the easiest thing to do here is to have a constant with fixed data at the top of the module, as we've done before.

File: eox_core/api/v1/tests/integration/test_views.py [hunk 22]
```diff
@@ -0,0 +1,434 @@
...
+        self.assertEqual(response_data["name"], updated_data["fullname"])
+        self.assertEqual(response_data["mailing_address"], updated_data["mailing_address"])
+        self.assertEqual(response_data["year_of_birth"], updated_data["year_of_birth"])
+        self.assertEqual(response_data["gender"], updated_data["gender"])
+        self.assertEqual(response_data["level_of_education"], updated_data["level_of_education"])
+        self.assertEqual(response_data["goals"], updated_data["goals"])
+        self.assertFalse(response_data["is_active"])
+        # self.assertEqual(response_data["phone_number"], updated_data["phone_number"])
+        # self.assertEqual(response_data["country"], updated_data["country"])
+        # self.assertEqual(response_data["bio"], updated_data["bio"])
```
Comment (mariajgrimaldi) [inline eox_core/api/v1/tests/integration/test_views.py:380]: Do we need these lines?

File: setup.cfg (hunk 17 above)
[the remark this answers is not in the document]
Comment (mariajgrimaldi) [reply setup.cfg]: Looks good! Thank you

File: eox_core/api/v1/tests/integration/test_views.py (hunk 22 above)
Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py:380]: Well, I wouldn't think so. But since you remind me, it would be good to check why those fields are not being updated. I'm really not sure if that should be the expected behavior.

File: eox_core/api/v1/tests/integration/test_views.py (hunk 20 above)
[the remark this answers is not in the document]
Comment (magajh) [reply eox_core/api/v1/tests/integration/test_views.py:16]: @BryanttV I'm not 100% sure what the best approach should be here. I'd suggest adding the constant at the beginning of the file, as @mariajgrimaldi suggests, for now, and we can have a discussion later about what the best option would be

File: eox_core/api/v1/tests/integration/test_views.py (hunk 22 above)
Comment (mariajgrimaldi) [reply eox_core/api/v1/tests/integration/test_views.py:380]: I don't know either. Maybe there's more information right in the code. You could do a quick check in case it's something we should fix soon.

Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py]: According to this setting, these fields should be updated. 
https://github.com/eduNEXT/eox-core/blob/a8ae7152a22084b7188e18550086da9d0e52cbb7/eox_core/settings/common.py#L48

**Comment by BryanttV** [comment]
Hi @magajh @mariajgrimaldi. I had to create a new PR because this one was too long according to the label workflow. We should merge this one first: https://github.com/eduNEXT/eox-core/pull/279

**Review by magajh — APPROVED** [review]
Thanks, @BryanttV 
LGTM

File: eox_core/api/v1/tests/integration/test_views.py (hunk 22 above)
Comment (mariajgrimaldi) [reply eox_core/api/v1/tests/integration/test_views.py]: Can you replicate this elsewhere? Can we create an internal ticket to address this issue?

**Review by mariajgrimaldi — APPROVED** [review]
Looks good, thank you!

File: eox_core/api/v1/tests/integration/test_views.py (hunk 22 above)
Comment (BryanttV) [reply eox_core/api/v1/tests/integration/test_views.py]: I just tested on the Redwood stage and the same thing happens. I will open an issue

**Review by magajh — APPROVED** [review]
